### DISCUSSION TITLE
roboshop

### DISCUSSION THREAD

**Discussion by GCKeita** [learndevopsonline/learndevopsonline#1925] [Q&A] [answered]
Can anyone assist plz? getting an errors running script
1- On redis
error code= 
${color}Updating Redis Listen Address
FAILURE

hint: line in the script=
echo -e "\${color}Updating Redis Listen Address${nocolor}"
sed -i -e 's/127.0.0.1/0.0.0.0' /etc/redis.conf /etc/redis/redis.conf &>>/tmp/roboshop.log
stat_check $?

2- On mysqlerror code=
 Copying MySQL Repo File 
FAILURE

hint:
echo -e "${color} Copying MySQL Repo File ${nocolor}"
cp /root/roboshop-shell/roboshop-shell/mysql.repo /etc/yum.repos.d/mysql.repo &>>/tmp/roboshop.log
stat_check $?

**Comment by chaithanya1812** [comment, marked as answer]
Hello @GCKeita 
### In sed command '/' --> slash is missing 
## sed -i -e 's/127.0.0.1/0.0.0.0/'  /etc/redis.conf /etc/redis/redis.conf &>>/tmp/roboshop.log

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/27de1cd4-0e35-4700-a881-185e82199f7f)

**Comment by chaithanya1812** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/9d639830-1f7e-4dda-b8ae-a3ac1040b1fe)

**Comment by chaithanya1812** [comment]
Hello @GCKeita 
### Don't waste time on re-running the code which you written 5 months ago,
### Start fresh one!.. watch the lectures again & build the new code. It will be happy path.

**Reply by GCKeita** [reply]
Ok 
But I am not sure to have access to new lectures

**Reply by chaithanya1812** [reply]
Not new lectures, Do revision of your batch videos from starting.

**Reply by GCKeita** [reply]
OK
